tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -61,6 +62,12 @@ def test_rbd_space_reclaim_cronjob(self):
         7. Verify the decreased used size of the RBD pool
         8. Verify the presence of other files in the folder
         """
+
+        # Wait for 45 seconds for cluster load to pause effectively
+        wait_time = 45
+        log.info(f"Waiting for {wait_time} seconds...")
+        time.sleep(wait_time)
```

Comment:
How about adding this in the fixture pause_and_resume_cluster_load ?

Reply:
Done